invenio_app_rdm/theme/assets/semantic-ui/less/invenio_app_rdm/theme/elements/icon.overrides

```diff
@@ -3,7 +3,8 @@
 ***********************************************/
 
 .inline-id-icon {
-  height: @fontSizeBase;
+  height: 1rem;
```

Comment:
@jennur isn't it better to use a variable here?

Reply:
I can't really say, as I don't know the reason for this change, but I'm guessing there is a reason (@lhenze )? In any case I think the value of the variable is the same, so not sure if it matters.

Reply:
Edit: Actually, the `@fontSizeBase` is 14px, so imo having it be 1rem is better to follow the settings of the browser.

Reply:
The ORCID requirement is "Scale the iD icon to match the line height of the surrounding text but go no smaller than 16×16." so maybe 1em would actually be correct?  (Rem seems to render fine though currently, I don't think the icon appears in any large header text.)

Reply:
The ORCID requirement is "Scale the iD icon to match the line height of the surrounding text but go no smaller than 16×16." so maybe 1em would actually be correct?  (Rem seems to render fine though currently, I don't think the icon appears in any large header text.)